src/Microsoft.ML.Transforms/ExpressionCatalog.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML
+{
+    public static class ExpressionCatalog
+    {
+        /// <summary>
+        /// Creates an <see cref="ExpressionEstimator"/>.
+        /// </summary>
+        /// <param name="catalog">The <see cref="TransformsCatalog"/>.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.
+        /// This column's data type will be the same as that of the input column.</param>
+        /// <param name="expression">The expression to apply to <paramref name="inputColumnNames"/> to create the column <paramref name="outputColumnName"/>.</param>
+        /// <param name="inputColumnNames">The names of the input columns.</param>
```

Comment:
> /// <param name="inputColumnNames">The names of the input columns.</param> [](start = 7, length = 75)

please add link to the sample

Reply:
Resolved in follow-up PR: https://github.com/dotnet/machinelearning/pull/4614/files#r362319272

#Resolved